src/telem/lib/input/mekanism/FusionReactorInputAdapter.lua

```diff
@@ -36,16 +36,21 @@ function FusionReactorInputAdapter:beforeRegister ()
         --     environmental_loss       = fn():call('getEnvironmentalLoss'),
         -- },
         coolant = {
+            water                       = fn():call('getWater'):div(1000):fluid(),

```

Comment:
These need to be modified to pull `amount` from the result of the call before calculating. See example from other adapters below:

https://github.com/cyberbit/telem/blob/f7e54a8d6cb496040cb7d15f99a7a0231a409fab/src/telem/lib/input/mekanism/PressurizedReactionChamberInputAdapter.lua#L19